### PR TITLE
Cookiecutter

### PR DESCRIPTION
**Pull request by sabiharustam**
set up cookiecutter cms style and html doc page. Still need to include correct links to travis etc if we want and also need to create pages in documentation website.

### CODE REVIEW

**Review by VanNguyen23 — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
-# mm_2019_sss_3
-* Developing Metropolis Monte Carlo program
+geometry_analysis
+==============================
+[//]: # (Badges)
```
Comment (b-butler) [inline README.md:3]: Remove badges.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
-# mm_2019_sss_3
```
Comment (b-butler) [inline README.md:1]: Shouldn't we want to have our project name here?

File: README.md [hunk 3]
```diff
@@ -1,2 +1,18 @@
...
+[//]: # (Badges)
+[![Build Status](https://travis-ci.org/sabiharustam/analysis_package.svg?branch=master)](https://travis-ci.org/sabiharustam/analysis_package)
+[![AppVeyor Build status](https://ci.appveyor.com/api/projects/status/REPLACE_WITH_APPVEYOR_LINK/branch/master?svg=true)](https://ci.appveyor.com/project/REPLACE_WITH_OWNER_ACCOUNT/geometry_analysis/branch/master)
+[![codecov](https://codecov.io/gh/sabiharustam/analysis_package/branch/master/graph/badge.svg)](https://codecov.io/gh/sabiharustam/analysis_package)
+
+TA python package for the MolSSI Software Summer School.
+
+### Copyright
+
+Copyright (c) 2019, Sabiha Rustam
```
Comment (b-butler) [inline README.md:12]: All our names go here.

File: appveyor.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
+environment:
```
Comment (b-butler) [inline appveyor.yml:1]: We don't need this if we aren't implementing CI

File: devtools/README.md [hunk 5]
```diff
@@ -0,0 +1,60 @@
+# Development, testing, and deployment tools
```
Comment (b-butler) [inline devtools/README.md:1]: Only need this file if we are using devtools

File: devtools/conda-envs/test_env.yaml [hunk 6]
```diff
@@ -0,0 +1,18 @@
+name: test
```
Comment (b-butler) [inline devtools/conda-envs/test_env.yaml:1]: Only need if using devtools

File: devtools/scripts/create_conda_env.py [hunk 7]
```diff
@@ -0,0 +1,95 @@
+import argparse
```
Comment (b-butler) [inline devtools/scripts/create_conda_env.py:1]: Same as other files.

File: devtools/travis-ci/before_install.sh [hunk 8]
```diff
@@ -0,0 +1,41 @@
+# Temporarily change directory to $HOME to install software
```
Comment (b-butler) [inline devtools/travis-ci/before_install.sh:1]: Don't need without CI.

File: docs/README.md [hunk 9]
```diff
@@ -0,0 +1,18 @@
+# Compiling geometry_analysis's Documentation
```
Comment (b-butler) [inline docs/README.md:1]: Change to our project name.

File: docs/conf.py [hunk 10]
```diff
@@ -0,0 +1,166 @@
...
+#
+# import os
+# import sys
+# sys.path.insert(0, os.path.abspath('.'))
+
+
+# -- Project information -----------------------------------------------------
+
+project = 'mcpy'
+copyright = ("2019, Sabiha Rustam. Project structure based on the "
```
Comment (b-butler) [inline docs/conf.py:23]: All our names go here.

File: mcpy/__init__.py [hunk 11]
```diff
@@ -0,0 +1,15 @@
+"""
```
Comment (b-butler) [inline mcpy/__init__.py:1]: This file should not have been overwritten.

Comment (b-butler) [reply mcpy/__init__.py:1]: This needs to be checkout individually from master.

**Review by b-butler — CHANGES_REQUESTED** [review]

**Pull request merged**